pkg/controller/jobframework/multikueue_test.go

```
@@ -0,0 +1,142 @@
+			adapter.EXPECT().GVK().Return(batchv1.SchemeGroupVersion.WithKind("Job")).AnyTimes()
+
+			remoteClient := client.Client(fake.NewClientBuilder().WithScheme(scheme).WithObjects(tc.remoteObjects...).Build())
+			if tc.remoteClient != nil {
+				remoteClient = tc.remoteClient(scheme, tc.remoteObjects...)
+			}
+
+			if tc.wantDeleted {
+				adapter.EXPECT().DeleteRemoteObject(gomock.Any(), gomock.Any(), gomock.Any(), key).Return(nil).Times(1)
+			}
+
+			err := jobframework.DeleteRemoteObjectIfOwned(context.Background(), localClient, remoteClient, adapter, key, origin)
```

golangci-lint (2.12.2)

[error] context.Background() could be replaced by t.Context() in TestDeleteRemoteObjectIfOwned

(usetesting)